src/App.js

```diff
@@ -1,15 +1,61 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
 
-import Navbar from '../src/components/Navbar/Navbar'
-import Map from '../src/components/Map/Map'
+import Map from '../src/components/Map/Map';
+import Modal from '../src/components/Modal/Modal';
+import Navbar from '../src/components/Navbar/Navbar';
```

Comment:
Useless path segments for "../src/components/Map/Map", should be "./components/Map/Map"